app/index.html

```diff
@@ -32,9 +32,204 @@ <h1 class="center-text primary-text-color main-heading">Hey there. I&apos;m Mari
+        </div>
+
+        <div class="work-item">
+            <div class="work-item-details">
+                <h2 class="work-item-title primary-font primary-text-color">Multi-Post Stories <br> Gain+Glory</h2>
+                <div class="work-item-lang primary-font">
+                    <div class="lang-item">
+                        <p>Ruby on rails</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>css</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>Javascript</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>html</p>
+                    </div>
+                </div>
+                <div>
+                    <button class="button work-item-button primary-font">See Project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="work-item">
+            <div class="work-item-details">
+                <h2 class="work-item-title primary-font primary-text-color">Multi-Post Stories <br> Gain+Glory</h2>
+                <div class="work-item-lang primary-font">
+                    <div class="lang-item">
+                        <p>Ruby on rails</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>css</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>Javascript</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>html</p>
+                    </div>
+                </div>
+                <div>
+                    <button class="button work-item-button primary-font">See Project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="work-item">
+            <div class="work-item-details">
+                <h2 class="work-item-title primary-font primary-text-color">Multi-Post Stories <br> Gain+Glory</h2>
+                <div class="work-item-lang primary-font">
+                    <div class="lang-item">
+                        <p>Ruby on rails</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>css</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>Javascript</p>
+                    </div>
+                    <div class="lang-item">
+                        <p>html</p>
+                    </div>
+                </div>
+                <div>
+                    <button class="button work-item-button primary-font">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section>
+        <section id="about" class="content">
+            <img class="about-bg-1" src="./assets/i_about_me_t.png" alt="">
+            <img class="about-bg-2" src="./assets/i_about_me_b.png" alt="">
```

Comment:
- [ ] I think these images are meant to be proper background image (decorations are always background images, not img tags). What do you think 🤔 ?

Reply:
[changes done] decorations are now background images